core/tauri-macros/src/command/wrapper.rs

```diff
@@ -187,23 +190,18 @@ fn parse_arg(command: &Ident, arg: &FnArg) -> syn::Result<TokenStream2> {
   )))
 }
 
-fn snake_case_to_camel_case(s: String) -> String {
-  if s.as_str().contains('_') {
-    let mut camel = String::with_capacity(s.len());
-    let mut to_upper = false;
-    for c in s.chars() {
-      match c {
-        '_' => to_upper = true,
-        c if to_upper => {
-          camel.push(c.to_ascii_uppercase());
-          to_upper = false;
-        }
-        c => camel.push(c),
-      }
-    }
+/// Convert a snake_case string into camelCase, no underscores will be left.
+fn snake_case_to_camel_case(key: &str) -> String {
+  let mut camel = String::with_capacity(key.len());
+  let mut to_upper = false;
 
-    camel
-  } else {
-    s
+  for char in key.chars() {
+    match char {
```

Comment:
```suggestion
  for c in key.chars() {
    match c {
```

using char as a binding makes syntax highlighting look a bit weird